test/types/index.test-d.ts

```diff
@@ -0,0 +1,19 @@
+import build, { OnUnknown } from "../../index";
+import { expectType } from "tsd";
+import { Duplexify } from "duplexify";
```

Comment:
Do not rely on Duplexify types. Using Duplexify is an implementation detail.

Reply:
I've removed the dependency on Duplexify and changed the type to Transform.